pkgs/applications/misc/rsclock/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname= "rsClock";
+  version= "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "valebes";
+    repo = pname;
+    rev ="49335356cd6d0544491cd97ede52e7e0ec027da2";
+    sha256 = "1fpidswkgpf9yr4vxqn38livz6r3z5i0lhg7ngj9f1ki4yqxn9zh";
+  };
+
+  checkPhase = "";    
```

Comment:
If you need to disable tests, please use ``doCheck = false;``.

Is there any reason they are not running?

Reply:
This package don't have any tests.